NAMING_CONVENTIONS.md

```diff
@@ -0,0 +1,37 @@
+Arnold USD naming conventions
+=======================
+
+The Arnold USD implementation supports arnold-specific attributes through custom schemas.
+
+## Arnold attributes on USD nodes
+
+Additional arnold attributes on USD-native nodes (geometries, lights, etc...) need to be in the "arnold" namespace
+
+Example:
+def DomeLight "my_light"
+{
+ float intensity=2
+ int arnold:max_bounces=99
+}
+
+## Arnold Shaders
+
+Arnold shaders are supported with UsdShade Shader primitives. The info:id attribute should be the shader name, with an "arnold" namespace. The attributes will be in the "inputs" scope.
```

Comment:
Arnold shaders are supported as UsdShade Shader primitives. The `info:id` attribute should be the Arnold shader name, prefixed by the `arnold` namespace. The shader parameter are placed in the `inputs` scope.